css/style.css

```diff
@@ -0,0 +1,81 @@
+@font-face {
+  font-family: poppins;
+  src: url("https://fonts.googleapis.com/css?family=Poppins");
+}
+
+.header {
+  width: 375px;
+  height: 90px;
+  color: #6070ff;
+  margin: auto;
+  display: flex;
+  justify-content: space-between;
+}
+
+h2 {
+  font-size: 25px;
+}
+
+.material-icons {
+  margin-top: 25px;
+  font-size: 35px;
+}
+
+.container {
+  width: 375px;
+  height: 718px;
+  margin: auto;
+  background-image: url(header-shapes\ mobile@2x.png);
+  background-repeat: no-repeat;
+  border-radius: 0 0 10px 0;
+}
+
+h1 {
+  font-size: 35px;
+  padding-top: 80px;
+  color: #172b4d;
+}
```

Comment:
- [ ] As mentioned above 👆 I think it would be better to use the given styles for getting a design close to the Figma template as much as possible. Therefore, please make sure to increase the `font-size` and `font-weight` according to the given design